### DISCUSSION TITLE
Aspire?

### DISCUSSION THREAD

**Discussion by chrisrlewis** [jasontaylordev/CleanArchitecture#1056] [General]
The most interesting news from the dotnetconf '23 for me was Aspire. I realise this is early days, and the announcement wasn't widely known about but are there plans to incorporate Aspire orchestration, deployment, etc. into this template?

**Comment by punkouter26** [comment]
me too. I am hoping go get some good demos of aspire and make it my starting point.. So would like to see it in this project

**Comment by melotic** [comment]
Especially with the release of Aspire 9, I think it's production ready. Would love to see it implemented here.

**Comment by jasontaylordev** [comment]
See #1274

**Comment by jasontaylordev** [comment]
I've added support for Aspire. Give it a try by running:
```
dotnet new install Clean.Architecture.Solution.Template
dotnet new ca-sln --help
```

You can add .NET Aspire 9 support with the `-use-aspire` command. For example:
```
dotnet new ca-sln --client-framework none --database postgresql --use-aspire --output CaAspireExample
```

Both the SQL Server and PostgreSQL Aspire integrations are supported.

Feel free to let me know if you have any questions.
